keyboards/inett_studio/sq80/info.json

```diff
@@ -0,0 +1,320 @@
+                { "matrix": [2, 7], "x": 8.5, "y": 2.5 },
+                { "matrix": [2, 8], "x": 9.5, "y": 2.5 },
+                { "matrix": [2, 9], "x": 10.5, "y": 2.5 },
+                { "matrix": [2, 10], "x": 11.5, "y": 2.5 },
+                { "matrix": [2, 11], "x": 12.5, "y": 2.5 },
+                { "matrix": [2, 12], "w": 1.5, "x": 13.5, "y": 2.5 },
+                { "matrix": [2, 13], "x": 15.25, "y": 2.5 },
+                { "matrix": [2, 14], "x": 16.25, "y": 2.5 },
+                { "matrix": [4, 14], "x": 17.25, "y": 2.5 },
+
+                { "matrix": [5, 2], "w": 1.75, "x": 0, "y": 3.5 },
+                { "matrix": [3, 0], "x": 1.75, "y": 3.5 },
+                { "matrix": [3, 1], "x": 2.75, "y": 3.5 },
+                { "matrix": [3, 2], "x": 3.75, "y": 3.5 },
+                { "matrix": [3, 3], "x": 4.75, "y": 3.5 },
+                { "matrix": [3, 4], "x": 5.75, "y": 3.5 },
+                { "matrix": [3, 5], "x": 6.75, "y": 3.5 },
+                { "matrix": [3, 6], "x": 7.75, "y": 3.5 },
+                { "matrix": [3, 7], "x": 8.75, "y": 3.5 },
+                { "matrix": [3, 8], "x": 9.75, "y": 3.5 },
+                { "matrix": [3, 9], "x": 10.75, "y": 3.5 },
+                { "matrix": [3, 10], "x": 11.75, "y": 3.5 },
+                { "matrix": [3, 11], "w": 2.25, "x": 12.75, "y": 3.5 },
+
+                { "matrix": [5, 3], "w": 2.25, "x": 0, "y": 4.5 },
+                { "matrix": [4, 0], "x": 2.25, "y": 4.5 },
+                { "matrix": [4, 1], "x": 3.25, "y": 4.5 },
+                { "matrix": [4, 2], "x": 4.25, "y": 4.5 },
+                { "matrix": [4, 3], "x": 5.25, "y": 4.5 },
+                { "matrix": [4, 4], "x": 6.25, "y": 4.5 },
+                { "matrix": [4, 5], "x": 7.25, "y": 4.5 },
+                { "matrix": [4, 6], "x": 8.25, "y": 4.5 },
+                { "matrix": [4, 7], "x": 9.25, "y": 4.5 },
+                { "matrix": [4, 8], "x": 10.25, "y": 4.5 },
+                { "matrix": [4, 9], "x": 11.25, "y": 4.5 },
+                { "matrix": [4, 10], "w": 2.75, "x": 12.25, "y": 4.5 },
+                { "matrix": [4, 13], "x": 16.25, "y": 4.5 },
+
+                { "matrix": [5, 4], "w": 1.25, "x": 0, "y": 5.5 },
+                { "matrix": [5, 5], "w": 1.25, "x": 1.25, "y": 5.5 },
+                { "matrix": [5, 6], "w": 1.25, "x": 2.5, "y": 5.5 },
+                { "matrix": [5, 7], "w": 6.25, "x": 3.75, "y": 5.5 },
+                { "matrix": [5, 8], "w": 1.25, "x": 10, "y": 5.5 },
+                { "matrix": [5, 9], "w": 1.25, "x": 11.25, "y": 5.5 },
+                { "matrix": [5, 10], "w": 1.25, "x": 12.5, "y": 5.5 },
+                { "matrix": [5, 11], "w": 1.25, "x": 13.75, "y": 5.5 },
+                { "matrix": [5, 12], "x": 15.25, "y": 5.5 },
+                { "matrix": [5, 13], "x": 16.25, "y": 5.5 },
+                { "matrix": [5, 14], "x": 17.25, "y": 5.5 }
+            ]
+        },
+        "LAYOUT_tkl_ansi_wkl": {
```

Comment:
```suggestion
        "LAYOUT_tkl_ansi_tsangan_wkl": {
```

Reply:
I'm sorry, I'm confused again.
Isn't this a textbook example of a WKL layout?

If we were to assume that I interpreted your other comment correct and that in QMK `tsangan` only refers to the bottom row. Then `tsangan_wkl` doesn't make sense, as it would be identical to `wkl`. Which is exactly what you are saying. But surely we should prefer the most fundamental layout and use `wkl` when referring to this layout?

If you really want me to, I will make the requested changes. But I can't help to feel like something isn't 100% right here.

Reply:
Is not an issue if you feel my suggestion doesn't feel right 🙂

To maybe help add some context to my suggestions, please do have a look at `tkl` layouts in `/layouts/default/` directory.

Again, suggestions of mine you're more than welcome to disregard 🙂

Reply:
@Wraul 
I'd be keen to hear your thoughts on PR 21809.